improver_tests/spotdata/spotdata/test_NeighbourSelection.py

```diff
@@ -502,12 +526,14 @@ def test_basic(self):
 
         plugin = NeighbourSelection()
         result, result_nodes = plugin.build_KDTree(self.region_land_mask)
-        expected_length = (self.region_land_mask.shape[0] *
-                           self.region_land_mask.shape[1])
+        expected_length = (
+            self.region_land_mask.shape[0] * self.region_land_mask.shape[1]
+        )
 
+        # pylint: disable=unsubscriptable-objec
         self.assertEqual(
-            result_nodes.shape[0],  # pylint: disable=unsubscriptable-object
-            expected_length)
+            result_nodes.shape[0], expected_length,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'result_nodes.shape' is unsubscriptable](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5354756)